internal/status/status.go

```diff
@@ -44,53 +47,65 @@ type statusResponse map[string]interface{}
 
 func GetStatus(w http.ResponseWriter, req *http.Request) {
```

Comment:
Now that this endpoint seems to be used officially in our code, it makes sense to promote it to a versioned api route (`/api/v1/status`) instead of `internal`.